.github/workflows/charts-release.yaml

```
@@ -86,20 +86,19 @@ jobs:
         run: |
           #!/bin/bash
           CHARTS=(${{ steps.collect-changes.outputs.modifiedChartsAfterTag }})
+          mkdir -p website/static/img/chart-icons || echo "chart-icons path already exists, continuing..."
           for i in "${CHARTS[@]}"
```

```suggestion
          mkdir -p website/static/img/chart-icons || echo "chart-icons path already exists, continuing..."
          rm -rf website/charts/*
          for i in "${CHARTS[@]}"
```
This builds docs for ALL apps not the changed only, right?
If so, we can delete everything under there `charts/`, and rebuild it. 
With this it will prune apps that are either moved to different train (eg incubator -> stable or vice verse) or removed completely

---

Yes, by moving the per-app docs to this repo and away from the website we can safely do this now :)